backend/src/v1/code-table/code-table.service.ts

```diff
@@ -540,6 +546,116 @@ export class CodeTableService {
         });
         return results;
       }
+      case "schedule": {
+        const { data } = await get(token, {
+          query: "{scheduleCodes{scheduleCode shortDescription longDescription displayOrder activeIndicator}}",
+        });
+        const results = data.scheduleCodes.map(
+          ({ scheduleCode, shortDescription, longDescription, displayOrder, activeIndicator }) => {
+            const table: Schedule = {
+              schedule: scheduleCode,
+              shortDescription: shortDescription,
```

Comment:
Probably being overly paranoid but given we know WebEOC does sometimes send over short descriptions beyond 50 characters I wonder is it worth putting in a check on this to avoid failing to pull over some of these tables? Backlog item perhaps?

Reply:
I noted this based upon CE-1000 but in checking with @marqueone-ps this doesn't work in the same way so wouldn't be affected by the 1000 problem. Moving to 'closed' comment